### DISCUSSION TITLE
File format for UniFormation GKtwo

### DISCUSSION THREAD

**Discussion by Choque** [sn4k3/UVtools#846] [Q&A] [answered]
I wanted to do the "per layer settings" test but for the life of me can't figure out which file format to use.

If I simply hit SAVE in UVTools, I can see the CTB file on my printer but can't start the print. It tries to validate the file, get's to around 60% and then throws some generic "File Error" error message.

If I do Convert to -> Chitubox -> Chitubox CTB for UniFormation GKtwo and try any of the versions from V1 through to V4, I don't even see the file on my printer UI.

Converting to JXS doesn't work either, since UVTools throws some "System.ArgumentOutOfRangeException: ResolutionX can not be different from 4920" error.

My printer is on the most recent V1.2.0-202310 firmware.

If I can provide anything to help troubleshot the problem, for example a CTB file of something I printed with the printer, I'll be happy to help.

**Comment by sn4k3** [comment]
Try to use the CTB for gktwo but then rename file to .jxs

**Comment by Choque** [comment]
Renamed v1 through v4 to .jxs, the result is the same as with .ctb though - none of the files show up for selection in the printer UI

**Reply by sn4k3** [reply]
Attach two files, one that work (original) and one that don't (same file processed by uvtools)

**Reply by Choque** [reply]
Here you go [GKtwo_Files.zip](https://github.com/sn4k3/UVtools/files/14531013/GKtwo_Files.zip)

**Reply by sn4k3** [reply]
The unmodified is CTB encrypted while modified is normal CTB.
Have you converted the unmodified? Save file would not convert file alone. But if you convert use CTB Encrypted instead

**Reply by Choque** [reply]
Yes, like I wrote, I've saved the file via Convert to -> Chitubox -> Chitubox CTB for UniFormation GKtwo. Was that the wrong way?

**Reply by sn4k3** [reply, marked as answer]
No, but maybe last firmware don't accept it anymore.
Try with: Convert to -> Chitubox -> Chitubox CTB (Encrypted).

But you don't need to convert file. You can use Save or Save As under original file.

**Reply by Choque** [reply]
Ok, just tested both.

Save/Save As: ❌ Doesn't work. File is visible on the printer but results in the generic "File Error" message when trying to start the print

Convert to -> Chitubox -> Chitubox CTB (Encrypted): ✅ Works. File shows up and I'm able to start the print.

I was also able to do the ["per layer settings" test](https://github.com/sn4k3/UVtools/wiki/Printer-compability-with-per-layer-settings-and-advanced-tools) with the file that I saved via Convert to -> Chitubox -> Chitubox CTB (Encrypted). So the GKtwo is compatible with the "per layer settings and advanced tools like Dynamic layer heights"

**Reply by sn4k3** [reply]
But the `File - Save` should work under the original file out of slicer (If they are the encrypted already).
What slicer and version you use?
